core/primitives/src/views.rs

```diff
@@ -1050,6 +1050,10 @@ pub enum ActionView {
     DeleteAccount {
         beneficiary_id: AccountId,
     },
+    Delegate {
+        delegate_action: DelegateAction,
+        signature: Signature,
+    },
```

Comment:
curious why some parts of the PR are gated behind the feature flag and not some things like this. Is this not an inconsistency or is there a reason for example the views here are consistent regardless of protocol feature?

Reply:
I tried to wrap the new code in the feature flag but this requires a lot of changes. Therefore I decided to return an error when `SignedDelegateAction` is submitted and manage it with the feature flag. I mentioned this in [the comment](https://github.com/near/nearcore/pull/7497#issuecomment-1317879732)